src/components/twilio-voice-dialer/public/twilio-voice-dialer.js

```diff
@@ -95,28 +105,18 @@ class TwilioVoiceDialer extends HTMLElement {
     this.#call.reject();
   }
 
-  #incomingHandler = (call) => {
-    this.#incomingEvent(call);
-
-    call.on('disconnect', this.#reset);
-    call.on('cancel', this.#reset);
-    call.on('reject', this.#reset);
-    this.#call = call;
-    this.#setStatus('incoming');
-  };
-
-  #incomingEvent(call) {
-    const incomingEvent = new CustomEvent('incoming', {
+  #raiseIncomingEvent(call) {
```

Comment:
Sorry for being so nitpicky: I see that we actually "dispatch" events, should we rename `raise` to `dispatch` instead? I didn't notice sooner, apologies.

Also, the events that we're creating with `new CustomEvent` should not have the labels `raiseX`, just `X`. Like on line 109, the variable should just be `incomingEvent` and not `raiseIncomingEvent`.

Reply:
no worries! agreed `dispatch` will match

nice catch. yes makes sense for CustomEvent to drop the prefix.